docs/quick-start.md

```diff
@@ -3,22 +3,378 @@ title: Getting Started
+spec:
+  package: "crossplane/stack-gcp:master"
+```
+
+Save the above as `stack.yaml`, and apply it by running:
+
+```bash
+kubectl apply -f stack.yaml
+```
+
+We've now installed Crossplane with GCP support! Take a look at the guide to
+[installing Crossplane] for more installation options, and to learn how to
+install support for other cloud providers such as Amazon Web Services and
+Microsoft Azure.
+
+## Configure Crossplane to Connect to GCP
+
+We've taught Crossplane how to work with GCP - now we must tell it how to
+connect to your GCP project. We'll do this by creating a Crossplane `Provider`
+that specifies the project name and some GCP service account credentials to use:
+
+```yaml
+apiVersion: gcp.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  name: example-provider
+spec:
+  # Make sure to update your project's name here.
+  projectID: my-cool-gcp-project
+  credentialsSecretRef:
+    name: example-gcp-credentials
+    namespace: crossplane-system
+    key: credentials.json
+```
+
+Save the above `Provider` as `provider.yaml`, save your Google Application
+Credentials as `credentials.json`, then run:
+
+```bash
+kubectl -n crossplane-system create secret example-gcp-credentials --from-file=credentials.json
+kubectl apply -f provider.yaml
+```
+
+Crossplane can now manage your GCP project! Your service account will need the
+CloudSQL Admin role for this guide. Check out GCP's [Getting Started With
+Authentication] guide if you need help creating a service account and
+downloading its `credentials.json` file, and Crossplane's [GCP provider
+documentation] for detailed instructions on setting up your project and service
+account permissions.
+
+## Provision a MySQL Instance
+
+GCP provides MySQL databases using [CloudSQL] instances. Crossplane uses a
+resource and claim pattern to provision and manage cloud resources like CloudSQL
+instances - if you've ever used [persistent volumes in Kubernetes] you've seen
+this pattern before. The simplest way to start using a new MySQL instance on GCP
+is to provision a `CloudsqlInstance`, then claim it via a `MySQLInstance`. We
+call this process _static provisioning_.
+
+
+```yaml
+apiVersion: database.gcp.crossplane.io/v1beta1
+kind: CloudsqlInstance
+metadata:
+  name: example-cloudsql-instance
+spec:
+  providerRef:
+    name: example-provider
+  writeConnectionSecretToRef:
+    name: example-cloudsql-connection-details
+    namespace: crossplane-system
+  forProvider:
+    databaseVersion: MYSQL_5_6
+    region: us-west2
+    settings:
+      tier: db-n1-standard-1
+      dataDiskType: PD_SSD
+      dataDiskSizeGb: 10
+      ipConfiguration:
+        ipv4Enabled: true
+```
+
+First we create a CloudSQL instance. Save the above as `cloudsql.yaml`, then
+apply it using `kubectl`:
+
+```bash
+kubectl apply -f cloudsql.yaml
+```
+
+Crossplane is now creating the `CloudsqlInstance`! Before we can use it, we need
+to claim it.
+
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
```

Comment:
these examples are a little less magic (in a good way) than the `cat` examples; esp. nice for readability when just scrolling through